File: src/main/java/com/projectincremental/documents/Personnage.java
package com.projectincremental.documents;

public class Personnage {

	private Long id;
	private String nom;
	private Long niveau;
	private String nomZone;
	
	public Long getId() {
		return id;
	}

	public void setId(Long id) {
		this.id = id;
	}

	public String getNom() {
		return nom;
	}

	public void setNom(String nom) {
		this.nom = nom;
	}

	public Long getNiveau() {
		return niveau;
	}

	public void setNiveau(Long niveau) {
		this.niveau = niveau;
	}

	public String getNomZone() {
		return nomZone;
	}

	public void setNomZone(String nomZone) {
		this.nomZone = nomZone;
	}

}

File: src/main/ui/src/Models/Personnage.js
class Personnage {
  constructor(nom, niveau, nomZone, caracteristics) {
    this.nom = nom;
    this.niveau = niveau;
    this.nomZone = nomZone;
    this.caracteristics = caracteristics;
  }

  set nomZone(nomZone) {
    this.nomZone = nomZone;
  }
}
